Apple/App/App-macOS.entitlements

```diff
@@ -8,7 +8,7 @@
 	</array>
 	<key>com.apple.security.application-groups</key>
 	<array>
-		<string>$(APP_GROUP_IDENTIFIER)</string>
+		<string>group.com.hackclub.burrow</string>
```

Comment:
We likely don't want this change – `APP_GROUP_IDENTIFIER` should be `group.com.hackclub.burrow`